SnipperAPI/Controllers/SnippetsControllers.cs

```diff
@@ -1,15 +1,32 @@
 ﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using SnipperAPI.interfaces;
+
 
 namespace SnippetApi.Controllers
 {
     [ApiController]
     [Route("[controller]")]
     public class SnippetsController : ControllerBase
     {
-        private static readonly List<Snippet> Snippets = new List<Snippet>();
+        private static readonly List<EncryptedSnippet> Snippets = new List<EncryptedSnippet>();
+
```

Comment:
Try to mind to keep on top of formatting, remove all the whitespace around your global vars. Will look at configuring your editor as there should be some formatting on save nice to haves which will avoid having to manually doing all this cleanup